scripts/bench_runner.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
```

Comment:
Is the bench_runner run as part of CI? How are we ensuring the benchmarks are met after changes to the code?